lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -19,7 +19,9 @@ def self.included(base)
 
       def setup_rabbitmq_connection_config
         # RabbitMQ server address
```

Comment:
Instead of having two settings for this, I propose just adding `:list => true` to the original `:host` setting. This will allow it to take multiple entries: `host => [ "host1", "host2" ]` in addition to the current behavior of single entry `host => "host1"`